unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -66,5 +70,25 @@ const LockMinorHeader = styled.div`
   color: var(--darkgrey);
 `
 
-const LockHeaderButton = styled.div`
+const LockHeaderButton = styled.div``
```

Comment:
Do we need this component at all since it does not have any style info an just has the `LockHeaderButtonContainer` inside it?